src/components/pages/GroomerProfile/GroomerProfileContainer.js

```diff
@@ -7,6 +9,23 @@ import { updateGroomer } from '../../../api/index';
 const GroomerProfileContainer = props => {
   const [contactModalVisible, setContactModalVisible] = useState(false);
   const [profileModalVisible, setProfileModalVisible] = useState(false);
+  const dummyData = {
```

Comment:
You should't need dummy data anymore. The groomer's info is retrieved from the redux store and passed as props.